packages/ui-components/src/components/search/usage/angular.md

```diff
@@ -0,0 +1,10 @@
+```html
+<!-- Default -->
+<kv-search></kv-search>
+
+<!-- Labeled -->
+<kv-search label="Text Field"></kv-search>
```

Comment:
don't think that this prop exists.

also it's missing the `placeholder` and `size` example